cmd/relay-gossip/README.md

```diff
@@ -138,7 +138,7 @@ func main() {
 
 #### With Known Chain Hash
 
-You do not need to know the full group info to use the pubsub client, if you know the chain hash and a HTTP endpoint then you can request the chain info from the HTTP endpoint, verifying it with the known chain hash:
+You do not need to know the full group info to use the pubsub client if you know the chain-hash and an HTTP endpoint then you can request the chain info from the HTTP endpoint, verifying it with the known chain hash:
```

Comment:
Why is chain hash hyphenated here?

Reply:
If it's different to a [hash chain](https://en.wikipedia.org/wiki/Hash_chain) then I think it should be hyphenated. If _hash chain_ and _chain hash_ are the same thing, then I suggest we use _chain hash_ across the docs.

Reply:
It is not hash chain 😄.

It's a hash of the information relating to the chain of randomness that's being generated by drand i.e. the public key, period and genesis time. I do not know if that necessitates a hyphen?

Reply:
Ok cool. So yeah you're right, doesn't need the hyphen. I think I threw another one in there somewhere. I'll find it and change it back.

Reply:
I've left the hyphen in `chain hash` on lines [21](https://github.com/drand/drand/pull/564/files#diff-bc07a701f06bee4379b40c8d8b9e47b6R21) and [31](https://github.com/drand/drand/pull/564/files#diff-bc07a701f06bee4379b40c8d8b9e47b6R31), it looks like they're addresses or something?